woocommerce/woocommerce
# Packages: Add Changelogger scripts

### All Submissions:






### Changes proposed in this Pull Request:

Add Changelogger to JS packages that didn't already have it. These changes only allow creation of the changelog entry, in a follow up PR I'll tackle creating the changelog.

### How to test the changes in this Pull Request:

1. `pnpm install`
2. Choose any js package that was brought over from WooCommerce Admin, such as `@woocommerce/tracks`
3. `pnpm nx changelog @woocommerce/tracks`
4. Follow the prompts and see a changelog entry created

### Other information:




### FOR PR REVIEWER ONLY:
